content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -82,17 +82,13 @@ packages that define the API objects.
 
 ### OpenAPI V3
 
-{{< feature-state state="beta"  for_k8s_version="v1.24" >}}
+{{< feature-state state="stable"  for_k8s_version="v1.27" >}}
 
-Kubernetes {{< param "version" >}} offers beta support for publishing its APIs as OpenAPI v3; this is a
-beta feature that is enabled by default.
-You can disable the beta feature by turning off the
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) named `OpenAPIV3`
-for the kube-apiserver component.
+Kubernetes {{< param "version" >}} supports publishing a description of its APIs as OpenAPI v3.
```

Comment:
Maybe we should avoid this shortcode and explicitly say that Kubernetes introduces support to OpenAPI v3 since v1.23 as an Alpha feature, the feature graduated into Beta stage in v1.24 and then GA'ed in 1.27.

The shortcode `{{< param "version" >}}` is a moving target. It gets new value when a new k8s version is out.
So it may causes confusion as if the feature is introduced just yesterday.

Reply:
Or we can say:

Kubernetes v1.27 provides Beta support to publish a description ...

Later on, in future release, we know that 1.27 is old enough. We will remove this version string entirely.

Reply:
Sorry I missed the notification earlier, updated thanks!